### ISSUE TITLE
Render busways

### ISSUE THREAD

**Issue by wmisener** [ZeLonewolf/openstreetmap-americana#308]
Busways, often parts of bus rapid transit systems, are roadways dedicated solely to buses and physically separated from other traffic. These are tagged as [highway=busway](https://wiki.openstreetmap.org/wiki/Tag%3Ahighway%3Dbusway), a relatively new tag, approved in April 2021. The tag has quickly gained popularity, rising to [>5000 uses](https://taginfo.openstreetmap.org/tags/highway=busway#chronology), though it has been hampered by [lack of rendering in OSM-Carto](https://github.com/gravitystorm/openstreetmap-carto/issues/4226#issuecomment-1116995998). In the US, such busways have existed for decades (e.g. in [Pittsburgh](https://en.wikipedia.org/wiki/South_Busway)), but are becoming more common, and now exist across the country:
![export](https://user-images.githubusercontent.com/58491489/167237686-d8ec80cd-1d09-4900-b4bf-48ac7a2a9056.png)

An easy initial approach would be to render them identically to service roads. This seems appropriate, as the busways don't form part of the regular road network, but they are important for orientation, as the map appears to have "gaps" without them:
<img width="367" alt="Screen Shot 2022-05-06 at 9 16 51 PM" src="https://user-images.githubusercontent.com/58491489/167237849-2452287f-b257-4711-9ca7-0137dcfba8e3.png">

Due to their importance, it might be good to have them render at an earlier zoom level than other service roads. Once access tagging is developed (#216), rendering similarly to roads with access=private is probably appropriate.

These busways will be more important once more public transport features are rendered. It might be nice in the future to use colors/styles that are in harmony with bus stops, or even with railway features (#101). But I think getting it on the map in some way and refining the rendering later is better than not rendering it at all.

**Comment by wmisener** [comment]
Does anyone have any city maps of Pittsburgh, or other cities with busways, that could be a useful inspiration for going beyond a simple service road depiction?

**Comment by 1ec5** [comment]
I found a couple that I posted in Slack [starting from about there](https://osmus.slack.com/archives/C01V02K52UX/p1630517383086000), but my collection doesn’t include many recent city-scale maps, so I’m also eager to see examples from others’ collections.

**Comment by Pengor** [comment]
There are some transit maps of Seattle prior to the removal of buses from the [Downtown Seattle Transit Tunnel](https://en.wikipedia.org/wiki/Downtown_Seattle_Transit_Tunnel) but those wouldn't really fit the style of this project so I won't bother linking them...

One idea for rendering would be placing a generic bus icon along busways to denote that they're only used by buses. If/when we start rendering specific transit routes then additional route numbers/shields could be placed alongside this "access" icon.

**Comment by ZeLonewolf** [comment]
In #318 we're toying with the idea of using purple as a generic color for public transportation infrastructure.  Perhaps in that vein we could take whatever we're using for `highway=service` and make it purple.

**Comment by claysmalley** [comment]
Guided busways (`highway=bus_guideway`) should be rendered along with unguided busways (`highway=busway`). `highway=busway` is present in OpenMapTiles but `highway=bus_guideway` is not, so this would require a change to OpenMapTiles.

**Comment by claysmalley** [comment]
[OpenMapTiles ticket for `highway=bus_guideway`](https://github.com/openmaptiles/openmaptiles/issues/1394)

**Comment by wmisener** [comment]
My first impression is that for an American style, these two tags should be rendered identically. Bus guideways are pretty rare in the US, and I don't know if the average American user would care about the difference. An overpass search reveals scattered use, but quick inspection reveals many of these don't look like true bus guideways. Some appear to be busways tagged for the renderer (i.e. for Carto), or simple bus turnarounds that might better be tagged as service. 
<img width="695" alt="Screen Shot 2022-05-25 at 8 01 55 AM" src="https://user-images.githubusercontent.com/58491489/170294942-3c5bf841-1d3f-4833-b7c2-7ab4a76dcab4.png">
In fact, the only correct use of highway=bus_guideway in the US might be the [Boring Company tunnels in Las Vegas](https://www.openstreetmap.org/way/948566155).

Then again, rendering them slightly differently could provide mapper feedback...

**Comment by bgo-eiu** [comment]
The criteria for what counts as a busway according to the documentation on the wiki are quite stringent, so I am curious to investigate how well the existing usage of the tag in the US (and potentially elsewhere) actually complies with it. I know at least locally I've never had an opportunity to use the tag even for physically separated, designated roads for buses because something disqualifies each of them. It might be that cyclists and pedestrians are allowed to use them between bus arrivals, that they connect to a road with a mixed traffic main carriageway, or even just that they still have to pull over from the main carriageway into a bay where the curb is on dedicated bus service ways.

Depending on what these networks look like, it may make sense to apply an even broader grouping that includes tags like bus=designated in addition to the ones mentioned above, especially if rendering busways results in isolated stretches of busway for which the connectivity isn't clear. Bus lanes parallel to the roadway are much more common in the US, and I would guess at least some of what is mapped as a busway merges with these despite the wiki advising against this. It's a little tricky to know at what zoom level and how a parallel bus lane would be rendered, but some kind of matching purple padding on roadways with bus lanes could help illustrate complete networks of separate ways and lanes exists without leaving seemingly isolated bus loops in places. This could also make it easier to distinguish the difference between ways which link dedicated lanes along main roads (common in downtowns), and ways which branch of directly from a main carriageway with mixed traffic (like school dropoff loops for buses).

This tool is a pretty good visualizer of bus lanes and various versions of bus-designated roadways that could be helpful for seeing what kind of ruleset would work for "busways" (including not really busways by the strict definition), and whether or not rendering parallel bus lanes in conjunction with them would be a good idea. https://zlant.github.io/bus-lanes/#13/39.3077/-76.6186

**Comment by bgo-eiu** [comment]
For what it's worth, I did check and the existing use of highway=busway in the US is more conservative than I assumed and for the most part does match most of the wiki's requirements. (With some random exceptions here and there.) All of them are quite small though and it's not apparent whether or not they merge with mixed traffic or a dedicated lane when they meet roadways.

![IMG_20220609_115023](https://user-images.githubusercontent.com/100172442/172893689-5b6e79c6-16ea-47cf-9622-98be8aa5401a.jpg)
Infographic I saw recently from the Maryland Transit Administration that has me thinking it's a good idea to show dedicated bus lanes on shared roads clearly. In a number of American cities, bus passengers make up the majority of road users at any given time, but few online maps account for that.

Baltimore was actually apparently the first city in the US to allow its transportation department to plan dedicated bus lanes, the original ones were removed in the 80s, but I'm curious to track down some maps that show that network as it could be an interesting reference point. I think there might actually be more precedent for showing bus lanes in print than on online maps from what I'm reading on the history of dedicated bus lanes, but I could be wrong about that.
